test/index-pipelines.test.js

```diff
@@ -67,35 +68,22 @@ describe('Index Pipeline Tests', () => {
           response: { result: { body: { docs: [] } } },
         };
       },
-    )({ version: '1.0.0', ...params }, 'test.html');
-    assert.equal(actionName, 'index-pipelines/html_json@1.0.0');
-  });
-  it('specifying an empty version omits the version tag', async () => {
```

Comment:
That test is no longer required, as the `index-pipelines` action is no longer a pipelined action.